python/lsst/daf/butler/registry/datasets/byDimensions/summaries.py

```diff
@@ -279,14 +279,14 @@ def update(
                     "dataset_type_id": dataset_type_id,
                     self._collectionKeyName: collection.key,
                 }
-                for dataset_type_id in dataset_type_ids
+                for dataset_type_id in sorted(dataset_type_ids)
             ],
         )
-        for dimension, values in summary.governors.items():
-            if values:
+        for dimension in sorted(summary.governors):
+            if values := summary.governors[dimension]:
```

Comment:
I think `for dimension, values in sorted(summary.governors.items()):` would have worked too, but no preference.

Reply:
Sorting the dimensions seemed more obvious/efficient to me than sorting tuples...